Kernel/Memory/MemoryManager.cpp

```diff
@@ -427,6 +427,12 @@ UNMAP_AFTER_INIT void MemoryManager::initialize_physical_pages()
                 highest_physical_address = range_end;
         }
 
+        if ((multiboot_flags & MULTIBOOT_INFO_FRAMEBUFFER_INFO) && !multiboot_framebuffer_addr.is_null() && multiboot_framebuffer_type == MULTIBOOT_FRAMEBUFFER_TYPE_RGB) {
```

Comment:
The thing I dislike about this is this very x86-specific because we have zero guarantee that we will use a multiboot bootloader on any other CPU architecture besides x86. Maybe put it with `#ifdef ARCH(X86_64)` then?